packages/suite/src/views/wallet/send/components/Outputs/index.tsx

```diff
@@ -78,38 +82,30 @@ export const Outputs = ({ disableAnim }: OutputsProps) => {
                             ease: motionEasing.transition,
                         }}
                     >
-                        <StyledCard>
+                        <StyledCard
```

Comment:
You can delete this:

```js
const StyledCard = styled(Card)`
    gap: ${spacingsPx.xs};
`;
```

and just use `<Card ... >...</Card>`

Reply:
fixed in 583730ab59d19f31b357cfcbf0972c12466f4e6d